Base/QTGUI/qSlicerWebWidget.h

```diff
@@ -112,6 +112,7 @@ public slots:
   /// If a file already exists at the provided file path, it will be overwritten.
   /// \sa QWebEnginePage::printToPdf
   void printToPdf(const QString& filePath);
```

Comment:
Nitpick: could you add doxyven group (something like @{ @}) around the documentation and declaration of these two methods?

Reply:
done